server/execution/services/entityloader.py

```diff
@@ -185,3 +194,10 @@ def load_execution(exec_id: int) -> bool:
         # Activate execution (makes it accessible by API)
         run.activate_execution(execution)
         return True
+
+
+def load_role(id: int) -> Role | None:
```

Comment:
is this maybe a duplicate to 139?

Reply:
139?

Reply:
line 139

Reply:
Ah true, I will make that function public